### PR TITLE
Persist editorstate

### PR DESCRIPTION
**Pull request by killjoy2013**
**Summary**

Below two utilities added;

```
convertFromRawToEditorState(editorStateRaw)
convertEditorStateToRaw(editorState)
```

### CODE REVIEW

**Comment by mrkev** [comment]
What's the context of this PR? Also worth noting, `fromJS` and `toJS` exist in `src/model/immutable/EditorState.js`.

**Comment by killjoy2013** [comment]
> What's the context of this PR? Also worth noting, `fromJS` and `toJS` exist in `src/model/immutable/EditorState.js`.

Hi,
This is for the issue 
https://github.com/facebook/draft-js/issues/2381

**Comment by killjoy2013** [comment]
@claudiopro @mrkev gentlemen, couldn't produce those travis ci build errors on my dev environment.
Using vscode 1.45.1 on windows 10 machine. 
```
Version: 1.45.1 (system setup)
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:27:35.169Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.14393
```
Installed **Flow Language Support** but it suffers Flow version 0.115.0  since we have ^0.125.0 in .flowconfig

**npm run build** never results with errors! 
What should I check?

**Comment by killjoy2013** [comment]
> @claudiopro @mrkev gentlemen, couldn't produce those travis ci build errors on my dev environment.
> Using vscode 1.45.1 on windows 10 machine.
> 
> ```
> Version: 1.45.1 (system setup)
> Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
> Date: 2020-05-14T08:27:35.169Z
> Electron: 7.2.4
> Chrome: 78.0.3904.130
> Node.js: 12.8.1
> V8: 7.8.279.23-electron.0
> OS: Windows_NT x64 10.0.14393
> ```
> 
> Installed **Flow Language Support** but it suffers Flow version 0.115.0 since we have ^0.125.0 in .flowconfig
> 
> **npm run build** never results with errors!
> What should I check?

**yarn run test-ci**  is doing exactly that 👍   learning...

File: src/model/encoding/__tests__/convertEditorStateToRaw-test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const updateSelection = new SelectionState({
+  anchorKey,
+  anchorOffset,
+  focusKey,
+  focusOffset,
+  isBackward,
+  hasFocus,
+});
+
+let newEditorState2 = EditorState.forceSelection(
```
Comment (claudiopro) [inline src/model/encoding/__tests__/convertEditorStateToRaw-test.js:40]: Nit: please use `const` if you're not reassigning this variable

File: src/model/encoding/convertFromRawToEditorState.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const ContentState = require('ContentState');
+const EditorState = require('EditorState');
+const SelectionState = require('SelectionState');
+const convertFromRawToDraftState = require('convertFromRawToDraftState');
+
+const convertFromRawToEditorState = (
+  rawEditorState: RawDraftEditorState,
+  decorator?: ?DraftDecoratorType,
+): EditorState => {
+  let contentState: ContentState = convertFromRawToDraftState(
```
Comment (claudiopro) [inline src/model/encoding/convertFromRawToEditorState.js:26]: Nit: these should all be `const`

File: examples/draft-0-10-0/playground/src/App.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ import {
   EditorState,
   convertFromHTML as convertFromHTMLClassic,
   convertToRaw,
-  convertFromRaw,
+  convertFromRaw
```
Comment (claudiopro) [inline examples/draft-0-10-0/playground/src/App.js:37]: Nit: can you restore this trailing comma?

**Comment by claudiopro** [comment]
Actually I stand corrected, @mrkev is right. The static methods `EditorState.fromJS()` and `.toJS()` already exist and do a lot more than what you need. Let us know if you have further questions!

**Comment by claudiopro** [comment]
The `fromJS()` is a static method of `EditorState`, so this is how you'd use it: https://codesandbox.io/s/draft-js-tofromjs-08w8c?file=/src/App.js

However, it looks like the `EditorState.fromJS()` API is really new and didn't make it to an npm release yet. I will publish one in the coming days unless @mrkev was planning to do so already.

**Comment by killjoy2013** [comment]
@claudiopro @mrkev 
I've tested **fromJS**. Works fine. Looks like we can achive editor persistency requirements this way.
Wish I knew it at the very beginning :-)
Anyway, if there's anything I can help, I'd be glad to do it. You can assign me what you consider appropriate.
Thanks again

**Pull request closed without merging**